pkgs/tools/misc/watchexec/default.nix

```diff
@@ -1,23 +1,30 @@
-{ stdenv, rustPlatform, fetchFromGitHub }:
+{ stdenv, lib, rustPlatform, fetchFromGitHub, CoreServices, CoreFoundation }:
 
-rustPlatform.buildRustPackage rec {
+with rustPlatform;
+
+buildRustPackage rec {
   name = "watchexec-${version}";
-  version = "1.9.0";
+  version = "1.8.6";
+  buildInputs = stdenv.lib.optional stdenv.isDarwin [ CoreServices ];
+
+  preConfigure = ''
+    export NIX_LDFLAGS="-F${CoreFoundation}/Library/Frameworks -framework CoreFoundation $NIX_LDFLAGS"
+  '';
```

Comment:
This should be made conditional on `stdenv.isDarwin`.

Reply:
```diff
-  preConfigure = ''
+  preConfigure = stdenv.lib.optionalString stdenv.isDarwin ''
```